arch/arm/soc/cypress/Kconfig

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2018, Cypress
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+choice
+prompt "Cypress PSoC6 MCU Selection"
+depends on SOC_SERIES_PSOC62
+
+config SOC_PSOC6_M0
+	bool "SOC_PSOC6_M0"
+	select CPU_CORTEX_M0PLUS
+	select CPU_CORTEX_M_HAS_VTOR
+
+config SOC_PSOC6_M4
+	bool "SOC_PSOC6_M4"
+	select CPU_CORTEX_M4
+
+endchoice
+
+config SOC_FAMILY_PSOC6
+	bool
+	# omit prompt to signify a "hidden" option
```

Comment:
Not sure that I understand correctly - should be config SOC_FAMILY_PSOC6... moved before the SOC_PSOC6_M0 and SOC_PSOC6_M4?

Reply:
Sorry, there was a typo in my comment - "commit" used instead of "comment".

And I grepped the exiting code, it has those lines in this order. So, sorry for the noise, nothing needs to be done.